schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+          },
+          {
+            "name": "chromosome",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_chromosome_str",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "chromosome",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "haploin_sufficiency",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "haploin_sufficiency",
+            "datatype": "string"
+          },
+          {
+            "name": "last_evaluated",
+            "datatype": "date"
+          },
+          {
+            "name": "clingen",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "triplo_sensitivity",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "triplo_sensitivity",
+            "datatype": "string"
+          },
+          {
+            "name": "last_evaluated",
+            "datatype": "date"
+          },
+          {
+            "name": "clingen",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_xref",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "db",
+            "datatype": "string"
+          },
+          {
+            "name": "id",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          },
+          {
+            "name": "url",
+            "datatype": "string"
+          },
+          {
+            "name": "status",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_status",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "status",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_gene_variant",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "gene_variant",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_names",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "names",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "type",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "hgvs_list",
```

Comment:
Same with `gene_list`, I think we could get rid of this table and unroll the `hgvs_ids` in-place in other tables.